libkbfs/init.go

```diff
@@ -616,11 +616,13 @@ func doInit(ctx Context, params InitParams, keybaseServiceCn KeybaseServiceCn,
 		log.Warning("Could not enable disk limiter: %+v", err)
 		return nil, err
 	}
+	goctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
Nit: why "goctx"?  It's not used in a goroutine.  Maybe `ctxWithTimeout` or `ctx10s`?